jaeger_client/span.py

```diff
@@ -151,45 +163,45 @@ def set_baggage_item(self, key, value):
             self.log_kv(key_values=logs)
         return self
 
-    def get_baggage_item(self, key):
+    def get_baggage_item(self, key: str) -> Optional[Any]:
         return self.context.baggage.get(key)
 
-    def is_sampled(self):
+    def is_sampled(self) -> bool:
         return self.context.flags & SAMPLED_FLAG == SAMPLED_FLAG
 
-    def is_debug(self):
+    def is_debug(self) -> bool:
         return self.context.flags & DEBUG_FLAG == DEBUG_FLAG
 
-    def is_rpc(self):
+    def is_rpc(self) -> bool:
         for tag in self.tags:
             if tag.key == ext_tags.SPAN_KIND:
                 return tag.vStr == ext_tags.SPAN_KIND_RPC_CLIENT or \
                     tag.vStr == ext_tags.SPAN_KIND_RPC_SERVER
         return False
 
-    def is_rpc_client(self):
+    def is_rpc_client(self) -> bool:
         for tag in self.tags:
             if tag.key == ext_tags.SPAN_KIND:
                 return tag.vStr == ext_tags.SPAN_KIND_RPC_CLIENT
         return False
 
     @property
-    def trace_id(self):
+    def trace_id(self) -> int:
         return self.context.trace_id
 
     @property
-    def span_id(self):
+    def span_id(self) -> int:
         return self.context.span_id
 
     @property
-    def parent_id(self):
+    def parent_id(self) -> int:
```

Comment:
Do you wanna do SpanContext first? Because parent_id can certainly be None
```
./jaeger_client/codecs.py:252:            parent_id = None
./jaeger_client/codecs.py:347:        if span_context.parent_id is not None:
```

Reply:
What do you mean with SpanContext first? Does it also need types?

Reply:
yes - you already had it in earlier PRs, but probably reverted?

Reply:
Let me add that